tensorboard/components/tf_runs_selector/tf-wbr-string.html

```diff
@@ -0,0 +1,65 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../tf-imports/polymer.html" />
+
+<!--
+  tf-wbr-string safely renders a string, with <wbr> elements inserted
+  around select delimiters.
```

Comment:
am I reading it correctly that wbr will be added at the end? Also, since delimiter is no longer configurable, can we just write out all the delimiter this would break? e.g., this will be "noop" for input like `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa` right?

Reply:
> am I reading it correctly that wbr will be added at the end?

Yes. (<https://github.com/tensorflow/tensorboard/pull/2586#discussion_r316451820>)

> Also, since delimiter is no longer configurable, can we just write out
> all the delimiter this would break?

Not sure what you mean—are you asking that we specify the delimiter
pattern in this doc comment? Sure, if you like.

> e.g., this will be "noop" for input like
> `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa` right?

Yes (as the previous implementation was).